packages/model-api-builder/README.md

```diff
@@ -0,0 +1,59 @@
+# @loopback/model-api-builder
+
+Types and helpers for packages contributing Model API builders.
+
+## Overview
+
+The `@loopback/model-api-builder` package provides a contract for extensions
+that contribute builders for repositories and controllers. Users can provide
+their model classes and the extensions provided will build their repository and
+controller based on the model.
+
```

Comment:
IIRC, this package has two aspects:
1. It allows extensions to contribute builders.
2. It's a booter that invokes the builder.

I feel the second aspect is more important to describe, because that's how most LoopBack users are going to use this extension.

How to contribute builders is something that extension developers will be interested in.

Having said that, documenting model api booter/builders properly is out of scope of this pull request, there is another user story for that - see https://github.com/strongloop/loopback-next/issues/2740. See also loosely related https://github.com/strongloop/loopback-next/issues/2740

I am proposing to make the following changes in the README:

- Inside `Overview`, prepend a new paragraph explaining that this extension allows LB applications to boot models declaratively from a model definition file and a model config file.
- Rework `Basic use` to aim at LB application developers, show an example how to use this new extension (enable the booter) in a LB application.
- Move the current "Basic use" content into a new section aimed at extension developers, called e.g. "Custom builders".

Reply:
Please disregard my comment above. I am in a rush to wrap up all tasks before I leave for my vacation and I did not realize that `model-api-builder` is describing the interface of model api _builders_, not the actual _booter_.

The current content in README is ok 👍